react-app/Makefile

```diff
@@ -45,6 +45,10 @@ typecheck:
 codegen:
 	yarn codegen
 
+.PHONY: test
+test:
+	yarn test
+
```

Comment:
could you help to add `test` to the ci phony above so we can help ci run the tests for us? 🙏

Reply:
done 👍